README.md

```diff
@@ -1,15 +1,47 @@
-# Whisper :dash:
+# Whisper :leaves:
 
 [![Version](https://img.shields.io/cocoapods/v/Whisper.svg?style=flat)](http://cocoadocs.org/docsets/Whisper)
 [![License](https://img.shields.io/cocoapods/l/Whisper.svg?style=flat)](http://cocoadocs.org/docsets/Whisper)
 [![Platform](https://img.shields.io/cocoapods/p/Whisper.svg?style=flat)](http://cocoadocs.org/docsets/Whisper)
 
+## Description
+
+Break the silence of your UI, whispering or shouting at it. **Whisper** is a component that will make the task of display messages and in-app notifications simple. It has two different views inside.
+
+#### Whispers
+
+![Whisper](https://github.com/hyperoslo/Whisper/blob/feature/README/Resources/permanent-whisper.png)
+
+Display a short message at the bottom of the navigation bar, this can be anything, from a "Great Job!" to an error message. It can have images or even a loader.
+
+#### Shouts
+
+![In-App](https://github.com/hyperoslo/Whisper/blob/feature/README/Resources/in-app-notification.png)
+
+Fully customizable, from colors to fonts. This view will have an optional action that will be called if the user taps the view.
+
+The factory will handle the inset of the scroll view and adjust it to fit the Whisper, if you move from one view controller to another, it will animate the next controller's offset too.
```

Comment:
`The factory will handle the inset of the scroll view and adjust it to fit the Whisper, if you move from one view controller to another, it will animate the next controller's offset too.`

```
There is no need to think about scroll view insets anymore, this will be handled automagically by Whisper.
As and added bonus, when transitioning from one view controller to another, the next controllers offset will be adjusted like you would except. It just works!
```